Button/button.css

```diff
@@ -0,0 +1,244 @@
+    /* color asset for success color */
+    --success-color:#28a745;
+    --success-colorborder:#28a745;
+    --success-hover:#218838;
+    --success-hoverborder:#1e7e34;
+    /* color asset for link button */
+    --link-color:#007bff;
+    --link-hover:#007bff;
+
+    /* Size values */
+    --padding-topnbottom:0.375rem;
+    --padding-leftright:0.75rem;
+    --font-size:1rem;
+    --border-radius:0.25rem;
+}
+/* Basic button styling starts Here */
+.btn{
+    display:inline-block;
+    margin-bottom: 0px;
+    font-weight:400;
+    text-align:center;
+    border-radius:var(--border-radius);
+    font-size:var(--font-size);
+    border:1px solid transparent;
+    padding:var(--padding-topnbottom) var(--padding-leftright);
+    line-height: 1.5;
+    cursor:pointer;
+    font-family: inherit;
+    white-space: nowrap;
+    vertical-align: middle;
+}
+
+.btn:hover{
+
+    transition: all 0.6s;
+}
+.btn-round{
+    
+    border-radius:50%;
+    min-height:80px;
+    min-width:80px;
+    font-size: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+
+
+
+.btn:active{
+    outline:none;
+    box-shadow:inset 0 3px 5px rgba(0 0 0 0.13)
+}
+.btn:hover{
+    color:var(--btn-hover);
+}
+.btn[disabled]{
+    
+    opacity:0.65;
+    box-shadow: none;
+    pointer-events: none;
+    /* This property I got to know now */
+    
+}
+/* large btn class */
+.btn-lg{
+    padding: 10px 16px;
+    font-size: 18px;
+    line-height: 1.3333333;
+    border-radius: 6px;
+}
+/* small btn class */
+.btn-sm{
+    padding: 5px 10px;
+    font-size: 12px;
+    line-height: 1.5;
+    border-radius: 3px;
+}
+/* default btn class */
+.btn-secondary{
+    color:var(--white-color);
+    background-color:var(--secondary-color);
+    border-color:var(--secondary-colorborder);
+}
+.btn-secondary:hover{
+    background-color: var(--secondary-hover);
+    background-color: var(--secondary-hoverborder);
+    color:var(--white-color);
+}
+/* primary btn class */
+.btn-primary{
+    color:var(--white-color);
+    background-color:var(--primary-color) ;
+    border-color:var(--primary-colorborder);
+}
+.btn-primary:hover{
+    color:var(--white-color);
+    background-color:var(--primary-hover);
+    border-color: var(--primary-hoverborder);
+}
+/* btn warnng class */
+.btn-warning{
+    color:var(--white-color);
+    background-color:var(--warning-color);
+    border-color:var(--warning-colorborder);
+}
+.btn-warning:hover{
+    color:var(--white-color);
+    background-color:var(--warning-hover) ;
+    border-color:var(--warning-hoverborder) ;
+    
+}
+/* btn info class */
+.btn-info {
+    color:var(--white-color);
+    background-color: var(--info-color);
+    border-color: var(--info-colorborder);
+}
+.btn-info:hover {
+    color:var(--white-color);
+    background-color: var(--info-hover);
+    border-color:var(--info-hoverborder) ;
+}
+/* btn-link class */
+.btn-link{
+    color:var(--link-color);
+    border-radius:0;
```

Comment:
Can we try removing this? Looks like it's a default value.

Reply:
Done